python/set_up_staging_workspace_and_dataset.py

```diff
@@ -0,0 +1,528 @@
+import json
+import logging
+import os
+from argparse import ArgumentParser, Namespace
+
+from typing import Optional
+
+from utils.tdr_utils.tdr_api_utils import TDR
+from utils.tdr_utils.tdr_ingest_utils import StartAndMonitorIngest
+from utils.terra_utils.terra_util import TerraWorkspace, TerraGroups
+from utils.terra_utils.terra_workflow_configs import WorkflowConfigs, GetWorkflowNames
+from utils.request_util import RunRequest
+from utils.token_util import Token
+from utils.gcp_utils import GCPCloudFunctions
+from utils import GCP, comma_separated_list
+
+logging.basicConfig(
+    format="%(levelname)s: %(asctime)s : %(message)s", level=logging.INFO
+)
+
+GROUP_MEMBER = "member"
+GROUP_ADMIN = "admin"
+OWNER = "OWNER"
+WRITER = "WRITER"
+READER = "READER"
+
+# Define the relative path to the file
+STAGING_WORKSPACE_DESCRIPTION_FILE = "../general_markdown/staging_workspace_description.md"
+WDL_READ_ME_PATH = "../wdl/{script_name}/README.md"
```

Comment:
Should this be an f-string?

Reply:
I don't think I can because this is used later on any amount of workflows, so we don't yet know the script name to fill in.